src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -46,9 +46,11 @@ const (
 	maxDNSServiceIPAttempts   = 45
 	KeepWaiting               = false
 	ExitWaiting               = true
+	customManifestsFile       = "custom_manifests.yaml"
 )
 
 var (
+	retryPostManifestTimeout = 5 * time.Minute
```

Comment:
enough? not sure how much time takes to olm though

Reply:
I've increased it, just to be sure.